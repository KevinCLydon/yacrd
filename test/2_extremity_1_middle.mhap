1 2 0.0 2 0 100 450 1000 0 550 900 1000
1 3 0.0 2 0 550 900 1000 0 100 450 1000

