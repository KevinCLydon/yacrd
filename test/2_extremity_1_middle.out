Chimeric:1,1000;100,450,550;
